tasks/bootstrap_user.yml

```diff
@@ -16,10 +16,17 @@
 - name: Set bootstrap user as Galaxy Admin
   lineinfile: dest={{ galaxy_config_file }} state=present insertafter="app:main" regexp="^[ ]*admin_users[ ]*=[ ]*" line="admin_users = {{ galaxy_tools_admin_user }}"  #"
   when: (create_user is defined) and create_user and (admin_users_found.msg == "")
+  notify:
+    - restart galaxy
```

Comment:
The handler task was already called "restart galaxy", but for ansible to automatically recognize "restart galaxy" as a handler it has to be in handlers/main.yml (or be included from handlers/main.yml)